src/installer/corehost/cli/hostpolicy/static/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+project(hostpolicy_static)
+
+set(DOTNET_PROJECT_NAME "hostpolicy_static")
+
+# Include directories
+include_directories(../../fxr)
+include_directories(../../json)
+
+# CMake does not recommend using globbing since it messes with the freshness checks
+set(SOURCES
+    ../args.cpp
+    ../breadcrumbs.cpp
+    ../coreclr.cpp
+    ../deps_resolver.cpp
+    ../hostpolicy_context.cpp
+    ../hostpolicy.cpp
+    ../hostpolicy_init.cpp
+    ../../bundle/dir_utils.cpp
+    ../../bundle/extractor.cpp
+    ../../bundle/file_entry.cpp
+    ../../bundle/manifest.cpp
+    ../../bundle/runner.cpp
+)
+
+set(HEADERS
```

Comment:
This naming of `static` and `shared` folders is a bit confusing. I'd generally expect most files to be in shared folder.
But this point is rather moot, given the inclination to leave source files in their original locations.